src/modules/enrollment/components/pages/EnrollmentCeAssessmentsPage.tsx

```diff
@@ -108,15 +115,16 @@ const EnrollmentCeAssessmentsPage = () => {
         title='Coordinated Entry Assessments'
         headerVariant='border'
         actions={
-          canEditCeAssessments ? (
+          canEditCeAssessments &&
+          !ceAssessmentFeature?.legacy && (
```

Comment:
Same question here, I'm a little confused on the logic if it's `!undefined` then we still show the button

Reply:
You're right, in all these cases we should be safe to return Not Found if the feature doesn't exist. I think I was trying to minimize the possible problems arising from this change - logic something along the lines of "we weren't resolving the feature before, and now we are, so if it's null and we can't resolve it, we should behave as we would have before when we never tried to resolve it." But you're right, I don't think that logic is correct. We should always be able to resolve these features.